examples/pytorch/language-modeling/train_api_s3_dataset.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p>@<a href="https://github.com/deepanker13" target="_blank">deepanker13</a>&nbsp; Could you set keys instead of dummy keys like this:</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># Need to set S3 ACCESS_KEY 
S3_ACCESS_KEY = ""

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/training-operator/pull/2006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
done

Reply:
<p>I meant that we should define vars here or dedicated block like this:</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># Need to set S3 credentials
s3_access_key = ""
s3_secret_key = ""
</pre><p> </p><p>And then, we should use those vars here:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">&nbsp;&nbsp;# it is assumed for text related tasks, you have 'text' column in the dataset.

&nbsp;&nbsp;# for more info on how dataset is loaded check load_and_preprocess_data function in sdk/python/kubeflow/trainer/hf_llm_training.py

&nbsp;&nbsp;dataset_provider_parameters=S3DatasetParams(

&nbsp;&nbsp;&nbsp;&nbsp;{

&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; "endpoint_url": "http://10.117.63.3",

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"bucket_name": "test",

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"file_key": "imdatta0___ultrachat_1k",

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"region_name": "us-east-1",

+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"access_key": s3_access_key,

+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"secret_key": s3_secret_key,

&nbsp;&nbsp;&nbsp;&nbsp;}

&nbsp;&nbsp;),
</pre>